packages/api/webpack.config.js

```diff
@@ -80,6 +80,7 @@ module.exports = {
     }
   },
   externals: [
+    'aws-sdk',
```

Comment:
Is this intentional?

Reply:
Talked about this in huddle - likely not, but doesn't matter in either way: f8804386f14